scripts/vote_2025_05_11.py

```diff
@@ -0,0 +1,200 @@
+"""
+Chorus One oracle rotation
```

Comment:
Chorus one new address
0x285f8537e1dAeEdaf617e96C742F2Cf36d63CcfB
will be posted on Twitter later